doc/reference/cmdline.rst

```diff
@@ -38,9 +53,28 @@ Running the server
 
     .. (nb: when and why?)
 
+.. option:: --upgrade-path <upgrade_path>
+
+   specify an additional upgrade path.
+
+.. option:: --load <modules>
+
+   comma-separated list of server-wide modules. Default is ``base,web``.
```

Comment:
More risky than helpful without explanation as to what that is.

Reply:
again, description was taken from config.py
```
 group.add_option("--load", dest="server_wide_modules", help="Comma-separated list of server-wide modules.", my_default='base,web')

```
Please advice how it should be adapted.

Reply:
Let's use the description from here https://github.com/odoo/odoo/blob/master/odoo/conf/__init__.py#L21